### PR TITLE
Smile 1289 extract author affiliations with language guesser

### PR DESCRIPTION
**Pull request by anetteOlli**

### CODE REVIEW

File: create-publication-from-scopus/build.gradle [hunk 1]
```diff
@@ -4,6 +4,8 @@ dependencies {
     api(libs.nva.datamodel) {
         exclude group: 'com.github.bibsysdev', module: 'nva-commons'
     }
+    implementation group: 'org.apache.tika', name: 'tika-langdetect', version: '1.28.1'
+
```
Comment (anetteOlli) [inline create-publication-from-scopus/build.gradle:8]: Kunne ikke ta nyeste versjon. Dokumentasjonen matcher ikke det som faktisk eksporteres. (pluss nyeste versjon kom ut i februar 2022, så kanskje det kommer en bugfix senere)

File: create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java [hunk 2]
```diff
@@ -424,12 +427,23 @@ private Contributor generateContributorFromAuthorTp(AuthorTp author, AuthorGroup
...
         var organizationNameOptional = getOrganizationNameFromAuthorGroup(authorGroup);
-        return organizationNameOptional.map(organizationName -> Map.of(getLanguageIso6391Code(),
+        return organizationNameOptional.map(organizationName -> Map.of(getLanguageIso6391Code(organizationName),
                                                                        organizationName));
     }
 
-    private String getLanguageIso6391Code() {
-        return "en";
+    private String getLanguageIso6391Code(String textToBeGuessedLanguageCodeFrom) {
+        var detector = new OptimaizeLangDetector().loadModels();
```
Comment (brinxmat) [inline create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java:435]: Optimaize is a bit corny 🌽

File: create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java [hunk 3]
```diff
@@ -424,12 +427,23 @@ private Contributor generateContributorFromAuthorTp(AuthorTp author, AuthorGroup
...
+        var result = detector.detect(textToBeGuessedLanguageCodeFrom);
+        return result.isReasonablyCertain()
+                   ? getIso6391LanguageCodeForSupportedNvaLanguage(result.getLanguage())
+                   : ENGLISH.getIso6391Code();
```
Comment (brinxmat) [inline create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java:439]: Occurs to me that we may have similar issues with "main language" for publications, though there, we have a different solution…

**Review by brinxmat — APPROVED** [review]

File: create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java (hunk 2 above)
Comment (anetteOlli) [reply create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java:435]: Yeah, I tried to check if the newer version of tika just had changed the spelling to Optimize, but it seems the whole package is gone. (despite this is being an copy-paste example from their 2.3.0 documentation)

**Review by gregorgabriel — APPROVED** [review]

**Pull request merged**